examples/asset-upload/tsconfig.json

```
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events", "@4tw/cypress-drag-drop"],
```

```suggestion
        "types": ["cypress"],
```

I don't think this change is needed as these packages are not installed on this example